Source/Urho3D/Network/Connection.cpp

```diff
@@ -491,7 +470,7 @@ void Connection::ProcessLoadScene(int msgID, MemoryBuffer& msg)
 
     // In case we have joined other scenes in this session, remove first all downloaded package files from the resource system
     // to prevent resource conflicts
-    auto* cache = GetSubsystem<ResourceCache>();
+    ResourceCache* cache = GetSubsystem<ResourceCache>();

```

Comment:
Can't this be auto, C++11 is already a requirement and most of the code tries to follow.